test/TesterInternal/StorageTests/Relational/SqlServerStorageTests.cs

```diff
@@ -148,15 +148,15 @@ public void Dispose()
 
 
         [SkippableTheory, ClassData(typeof(StorageDataSetGeneric<string, string>))]
-        [TestCategory("Functional"), TestCategory("Persistence"), TestCategory("MySql")]
+        [TestCategory("Functional"), TestCategory("Persistence"), TestCategory(VendorCategory)]
```

Comment:
nit picking, but since you are doing this change in many places: You can move the Persistence and VendorCategory TestCategory attributes to the class level, instead of repeating on a method by method basis (we are slowly moving towards that since we migrated from MSTest). Functional or BVT is OK to apply at the method level even if currently all methods are of the same category, but you never know if a new test method will also be of the same kind.
Just a suggestion, feel free to ignore if there were no other changes required